src/styles/controls.plugin.scss

```diff
@@ -0,0 +1,39 @@
+.wikia-jw-small-player-controls {
+	#player_smallPlayerControls {
+		fill: white;
+		left: calc(50% - 52.5px);
```

Comment:
`52.5px` looks like a magic value ;)
how about cover the whole player with this div and just position icons inside it using flexbox?